add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code does not include any unit tests.

Recommendations:
- **Add Unit Tests**: Use PHPUnit to create unit tests for the `BookRepository` class.
- **Example**:
  ```php
  class BookRepositoryTest extends \PHPUnit\Framework\TestCase {
      public function testAddBook() {
          $mockConnection = $this->createMock(mysqli::class);
          $repository = new BookRepository($mockConnection);
          $this->assertTrue($repository->addBook('Title', 'Author', 2023));
      }
  }
  ```